developer/src/kmc-package/src/compiler/package-validation.ts

```diff
@@ -13,6 +13,10 @@ const MODEL_ID_PATTERN_PACKAGE = /^[a-z_][a-z0-9_]*\.[a-z_][a-z0-9_-]*\.[a-z_][a
 // const MODEL_ID_PATTERN_TS      = /^[a-z_][a-z0-9_]*\.[a-z_][a-z0-9_-]*\.[a-z_][a-z0-9_]*\.model\.ts$/;
 // const MODEL_ID_PATTERN_PROJECT = /^[a-z_][a-z0-9_]*\.[a-z_][a-z0-9_-]*\.[a-z_][a-z0-9_]*\.model\.kpj$/;
 
+// "Content files" within the package should adhere to these pattern:
+const CONTENT_FILE_BASENAME_PATTERN = /^[a-z0-9_+.-]+$/;
```

Comment:
Ah, I didn't realize '+' were valid filename characters. But the keyboard repo has
```
/legacy/v/visual_media_tamil_modular/source/help/visual_media_tamil_modular+chart.pdf
```

Reply:
Indeed. There are many other legal chars and I cannot recall now why we chose to allow `+` but not, for example, `(`.